core/sail/fts/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/QueryContextIteration.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Ditto. Should just say "Copyright (c) 2015 Eclipse RDF4J contributors." Check the code template settings.